packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -105,6 +103,38 @@ const dropdownStyles: ComponentSlotStylesPrepared<DropdownPropsAndState, Dropdow
       ...transparentColorStyleObj,
       alignItems: 'center',
     }),
+    ...(p.inverted && {
+      backgroundColor: v.backgroundColorInverted,
```

Comment:
props name comes in the beginning of the token `invertedBackgroundColor` please refactor the names

Reply:
fixed